### ISSUE TITLE
Citizens management needs more increment buttons

### ISSUE THREAD

**Issue by Anahkiasen** [deathraygames/civ-clicker#50]
Currently citizens management is capped at +/-100 after which it goes to +∞. Would be nice to have increments such as +1k and such when available, as is possible for buildings.

**Comment by olleharstedt** [comment]
Agree. One solution: Let user choose which buttons to show? Have a list of buttons in settings, check the buttons you want to use.

**Comment by olleharstedt** [comment]
I added 1k to my fork, but this is still an issue.

**Comment by JoelAzaria** [comment]
Agree as well, very inconvenient.

Another solution option - instead of the +/- ∞ buttons should be +/- custom.  Custom value would be a field on each line so each "category" has it's own custom value (I move differing amounts of each type - Tanners and Smiths by the thousand.  Miners and Farmers by the 100k or 500k, Wood workers by 50 or 100k, Soldiers by 500k and Cavalry by 50k.
So a custom "custom field" would prevent mistakes.  Esp since you can't dismantle/destroy buildings once you add them.

This same scheme should work and be applied imho to the Buildings tab as well.

**Comment by olleharstedt** [comment]
Does this repo accept pull requests?

**Comment by Lukenickerson** [comment]
@olleharstedt -- As long as the PR is focused and doesn't radically alter the game, then yes. I don't check this repo too often though, so don't expect anything fast. 😅

**Comment by Lukenickerson** [comment]
Possible duplicate of #33 ?

**Comment by olleharstedt** [comment]
> @olleharstedt -- As long as the PR is focused and doesn't radically alter the game, then yes. I don't check this repo too often though, so don't expect anything fast. sweat_smile

Maybe we can add a project page in Phabricator or such, to discuss possible directions?

**Comment by Lukenickerson** [comment]
I'm not sure what Phabricator is. Discussion here on the issues works well enough for me.
